cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureTemplateBuilder.java

```diff
@@ -152,6 +154,30 @@ public String build(CloudContext cloudContext, DatabaseStack databaseStack) {
         }
     }
 
+    private String getSkuTier(AzureDatabaseServerView azureDatabaseServerView) {
+        String skuTier = azureDatabaseServerView.getSkuTier();
+        String skuName = azureDatabaseServerView.getSkuName();
+        if (Objects.isNull(skuTier)) {
+            if (Objects.isNull(skuName)) {
+                return null;
+            } else {
+                return getSkuTierFromSkuName(skuName);
+            }
+        } else {
+            return azureDatabaseServerView.getSkuTier();
+        }
+    }
+
+    private String getSkuTierFromSkuName(String skuName) {
+        if (Set.of("GP_Gen5_2", "GP_Gen5_4", "GP_Gen5_8", "GP_Gen5_16", "GP_Gen5_32").contains(skuName)) {
+            return "GeneralPurpose";
+        } else if (Set.of("MO_Gen5_2", "MO_Gen5_4", "MO_Gen5_8", "MO_Gen5_16", "MO_Gen5_32").contains(skuName)) {
+            return "MemoryOptimized";
```

Comment:
these values might be better in private static fields

Reply:
Do you mean the instance types as well or just the two skuTiers (GeneralPurpose, MemoryOptimized)?

Reply:
yes, all of them